core/client/src/main/java/alluxio/client/file/FileInStream.java

```diff
@@ -82,6 +82,7 @@
   private final boolean mShouldCachePartiallyReadBlock;
   /** Whether to cache blocks in this file into Alluxio. */
   private final boolean mShouldCache;
+  private long mShouldCacheTier;
```

Comment:
(1) could we javadoc this member (just like `mShouldCache`)?
(2) `mShouldCacheTier` by alluxio code convention, sounds like a boolean value;